doc/source/whatsnew/v0.18.0.txt

```diff
@@ -20,6 +20,21 @@ Check the :ref:`API Changes <whatsnew_0180.api>` and :ref:`deprecations <whatsne
 New features
 ~~~~~~~~~~~~
 
+.. _whatsnew_0180.enhancements.extractall:
+
+The ``.str.extractall`` method is added (:issue:`11386`).
+Unlike ``extract`` (which returns only the first match),
+
+.. ipython:: python
```

Comment:
not sure I understand. what exactly needs to be indented 2 spaces? In the other whatsnew/*.txt files,
- The python code blocks seem to be indented 3 spaces, and
- The text following `_whatsnew` blocks does not seem to be indented.